tableaudocumentapi/archivefile.py

```diff
@@ -0,0 +1,67 @@
+import contextlib
```

Comment:
hah, containerfile?  It's only a zip archive because of implementation details.  Ultimately, it's a container file format.